FluidNC/src/Maslow/Maslow.cpp

```diff
@@ -488,7 +490,21 @@ void Maslow_::reset_all_axis(){
 }
 // move pulling just two belts depending on the direction of the movement
 bool Maslow_::move_with_slack(double fromX, double fromY, double toX, double toY) {
-  setTargets(toX,toY, 0);
+  
+  //How big of steps do we want to take with each loop?
+  double stepSize = 0.01;
+
+  double currentXTarget = getTargetX();
+  double currentYTarget = getTargetY();
+
+  //Compute the direction in X and Y
+  int xDirection = currentXTarget - toX > 0 ? -1 : 1;
+  int yDirection = currentYTarget - toY > 0 ? -1 : 1;
+
+  setTargets(currentXTarget + xDirection * stepSize, currentYTarget + yDirection * stepSize, 0, true, true, false, false);
```

Comment:
The `true, true, false, false` at the end here is what turns off setting the targets for the lower belts while we are repositioning. They are optional arguments so if we don't pass them then setTargets works just the same as before and does all four arms.